pkg/customcost/ingestor.go

```diff
@@ -54,116 +58,175 @@ type CustomCostIngestor struct {
 	lastRun      time.Time
 	runs         int
 	creationTime time.Time
-	coverage     opencost.Window
+	coverage     map[string]opencost.Window
```

Comment:
stringly-typed keys are something I'm not a huge fan of -- what is the key supposed to represent here? If its a plugin, why not have one CustomCostIngestor per plugin?